post_office/settings.py

```diff
@@ -79,6 +79,10 @@ def get_batch_size():
     return get_config().get('BATCH_SIZE', 100)
 
 
+def get_celery_enabled():
+    return get_config().get('CELERY_TASKS_ENABLED', True)
```

Comment:
Can you rename this to `CELERY_ENABLED`?

Reply:
Done